bpf/bpf_host.c

```diff
@@ -1135,7 +1135,8 @@ int cil_from_netdev(struct __ctx_buff *ctx)
 {
 	__u32 __maybe_unused vlan_id;
 
-#if defined(ENABLE_NODEPORT_ACCELERATION) && defined(ENABLE_EGRESS_GATEWAY)
+#if defined(ENABLE_NODEPORT_ACCELERATION)
+#if defined(HAVE_ENCAP)
```

Comment:
Nit: Why splitting on two lines?

Reply:
When thinking of additional cases where we need to punt from XDP, it felt more natural to bring some hierarchy in and list the `HAVE_ENCAP` part as just **one** case. The `XFER_PKT_SNAT_DONE` check for instance doesn't really belong under `HAVE_ENCAP`. It's just there because right now we know that there's no other reasons why a packet should be punted.

Reply:
OK sounds good.
Nit: If you update the commit, you can switch to `#ifdef` instead of `#if defined()`.